bin/templates/scripts/cordova/lib/projectFile.js

```diff
@@ -40,13 +40,43 @@ function parseProjectFile (locations) {
     const xcodeproj = xcode.project(pbxPath);
     xcodeproj.parseSync();
 
+    const workspaceName =
+        fs.readdirSync(project_dir).find(d => d.includes('.xcworkspace')) || '';
+
+    const projectName = workspaceName.replace('.xcworkspace', '');
```

Comment:
I've noticed we have a [`getIOSProjectname` function in `cordova-common`](https://github.com/apache/cordova-common/blob/87167bd45670291b68e6e02852a6b4faf63ed247/src/ConfigChanges/ConfigFile.js#L233), maybe you could use it here?